spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -1300,6 +1300,202 @@ public void testRangeCopyOnWriteMergePartitionedSortedTable() {
     checkCopyOnWriteDistributionAndOrdering(table, MERGE, expectedDistribution, expectedOrdering);
   }
 
+  // ===================================================================================
+  // Distribution and ordering for merge-on-read DELETE operations with position deletes
+  // ===================================================================================
+  //
+  // UNPARTITIONED
```

Comment:
I think it does not matter whether the table is sorted or not for position deletes.

Reply:
Agreed. Position deletes require their own sort order by _file and _pos that we will always use.